superbit_lensing/shear_profiles/annular_jmac.py

```diff
@@ -37,39 +38,158 @@
                     help='Name of simulation run')
 parser.add_argument('-outdir', type=str, default=None,
                     help='Output directory')
+parser.add_argument('-truth_file', type=str, default=None,
+                    help='Truth file containing redshift information')
+parser.add_argument('-nfw_file', type=str, default=None,
+                    help='Theory NFW shear catalog')
 parser.add_argument('--overwrite', action='store_true', default=False,
                     help='Set to overwrite output files')
 parser.add_argument('--vb', action='store_true', default=False,
                     help='Turn on for verbose prints')
 
+
+def compute_shear_bias(profile_tab):
+    '''
+    Function to compute the max. likelihood estimator for the bias of a shear profile
+    relative to the input NFW profile and the uncertainty on the bias.
+
+    Saves the shear bias estimator ("alpha") and the uncertainty on the bias ("asigma")
+    within the meta of the input profile_tab
+
+    The following columns are assumed present in profile_tab:
+
+    :gtan:      tangential shear of data
+    :err_gtan:  RMS uncertainty for tangential shear data
+    :nfw_gtan:  reference (NFW) tangential shear
+
+    '''
+
+    assert isinstance(profile_tab,Table)
+
+    try:
+        T = profile_tab['mean_nfw_gtan']
```

Comment:
Since the code in `master` now uses `_mean` and `_err` as the col suffix, we should either make that consistent here or you should change the code in `shear_plots.py` (after a pull from `master`) to make sure everything on `master` runs correctly after the PR merge

Reply:
If you don't mind, I would like to change the code in `shear_plots.py` to match this new naming convention.

Reply:
Fine with me!